recipes/pyspoa/meta.yaml

```diff
@@ -9,13 +9,15 @@ package:
 source:
 - url: https://github.com/nanoporetech/pyspoa/archive/refs/tags/v{{ version }}.tar.gz
   sha256: {{ sha256 }}
-- url: https://github.com/rvaser/spoa/archive/refs/tags/4.0.8.tar.gz
-  sha256: 3fc7e837b1052bd470924e37e0ed9531e7c839cd14b400573171d3d46d3ed9cd
+- url: https://github.com/rvaser/spoa/archive/refs/tags/4.1.2.tar.gz
+  sha256: 5619c7f1a9ebb9a04550771d1834ff61c4aa377bea44fbd99912e4137b1ac1f7
   folder: src
 
 build:
-  number: 0
+  number: 1
   skip: True  # [py < 38]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x.x") }}
```

Comment:
please replace `name` by `pyspoa`

Reply:
Thanks for catching that. I've edit name to 'pyspoa' in my next commit.